lib/src/model/settings/general_preferences.dart

```diff
@@ -87,6 +92,7 @@ class GeneralPrefsState with _$GeneralPrefsState {
     themeMode: ThemeMode.system,
     isSoundEnabled: true,
     soundTheme: SoundTheme.standard,
+    volume: 0.5,
```

Comment:
Have you tested that the 0.5 default? How does it compare with the current `null` default?

Reply:
I can't tell them apart, so it should be about the same.